.github/workflows/test.yml

```
@@ -17,6 +17,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - ruby: 2.7
```

By the way, let's use Ruby 3!
```suggestion
          - ruby: 3.0
```

---

I actually wanted to but unfortunately it will fail :( 
```
 ~/c/g/graphql-anycable  ⎇ allow-graphql-1.12 ↑1  ruby --version
ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-linux]
 ~/c/g/graphql-anycable  ⎇ allow-graphql-1.12 ↑1  bundle
Your Gemfile lists the gem fakeredis (>= 0) more than once.
You should probably keep only one of them.
Remove any duplicate entries and specify the gem only once.
While it's not a problem now, it could cause errors if you change the version of one of them later.
Fetching gem metadata from https://rubygems.org/.........
Fetching https://github.com/guilleiguaran/fakeredis
google-protobuf-3.14.0-x86_64-linux requires ruby version < 2.8.dev, >= 2.3, which is incompatible with the current
version, ruby 3.0.0p0
 5  ~/c/g/graphql-anycable  ⎇ allow-graphql-1.12 ↑1
```

https://github.com/protocolbuffers/protobuf/issues/7922

---

Ah, protobuf again. Every year, every Ruby release, same story :smile: